query/queries_and_views.ipynb

```diff
@@ -854,6 +859,2063 @@
+       "      <td>prov</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1055</td>\n",
+       "      <td>True</td>\n",
+       "      <td>HELLER EYECARE INC</td>\n",
+       "      <td>WILMINGTON</td>\n",
+       "      <td>US</td>\n",
+       "      <td>http://terminology.hl7.org/CodeSystem/organiza...</td>\n",
+       "      <td>prov</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>1079</td>\n",
+       "      <td>True</td>\n",
+       "      <td>PCP90763</td>\n",
+       "      <td>RAYNHAM</td>\n",
+       "      <td>US</td>\n",
+       "      <td>http://terminology.hl7.org/CodeSystem/organiza...</td>\n",
+       "      <td>prov</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>1211</td>\n",
+       "      <td>True</td>\n",
+       "      <td>PCP124160</td>\n",
+       "      <td>HYDE PARK</td>\n",
+       "      <td>US</td>\n",
+       "      <td>http://terminology.hl7.org/CodeSystem/organiza...</td>\n",
+       "      <td>prov</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1295</td>\n",
+       "      <td>True</td>\n",
+       "      <td>EYE ASSOCIATES OF SOMERVILLE INC.</td>\n",
+       "      <td>SOMERVILLE</td>\n",
+       "      <td>US</td>\n",
+       "      <td>http://terminology.hl7.org/CodeSystem/organiza...</td>\n",
+       "      <td>prov</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  org_id  active                                       name        city  \\\n",
+       "0   1003    True  ASSOCIATES OF SOUTH SHORE DERMATOLOGY LLC      MILTON   \n",
+       "1   1055    True                         HELLER EYECARE INC  WILMINGTON   \n",
+       "2   1079    True                                   PCP90763     RAYNHAM   \n",
+       "3   1211    True                                  PCP124160   HYDE PARK   \n",
+       "4   1295    True          EYE ASSOCIATES OF SOMERVILLE INC.  SOMERVILLE   \n",
+       "\n",
+       "  country                                             system  code  \n",
+       "0      US  http://terminology.hl7.org/CodeSystem/organiza...  prov  \n",
+       "1      US  http://terminology.hl7.org/CodeSystem/organiza...  prov  \n",
+       "2      US  http://terminology.hl7.org/CodeSystem/organiza...  prov  \n",
+       "3      US  http://terminology.hl7.org/CodeSystem/organiza...  prov  \n",
+       "4      US  http://terminology.hl7.org/CodeSystem/organiza...  prov  "
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with query_engine.connect() as con:\n",
+    "    con.execute(\"\"\"\n",
+    "      CREATE OR REPLACE VIEW flat_organization AS\n",
```

Comment:
For Organization, can we also add:

* type
* partOf (ref another organization

Assumption for the last one is that it may be needed for drilling down into organization hierarchy